app/controllers/search_controller.rb

```diff
@@ -6,6 +6,25 @@ def results
     @results = Post.where('title LIKE ?', "%#{search_term}%")
                .or(Post.where('body LIKE ?', "%#{search_term}%"))
                .or(Post.where('username LIKE ?', "%#{search_term}%"))
+
+    case params[:feedback]
+    when "true positive"
+      feedback = 1
```

Comment:
These aren't guaranteed to always be the correct IDs - put them in the select tag instead and use that to send them across

Reply:
Hmm, makes sense. Thanks! I'll make the change right away.